fusionfrontend/src/app/ecosystem/components/content/asset-type-dialog/asset-type-dialog.component.ts

```diff
@@ -71,6 +72,7 @@ export class AssetTypeDialogComponent implements OnInit {
   private createAssetTypeForm(assetTypeToEdit: AssetTypeDetails) {
     this.assetTypeForm = this.formBuilder.group({
       id: [],
+      version: [],
```

Comment:
the version has to be added above in onSave, better use: this.assetTypeForm.getRaw() as AssetType